src/global/propagation.rs

```diff
@@ -23,20 +23,20 @@ lazy_static::lazy_static! {
 /// // assign it as the global propagator
 /// global::set_http_text_propagator(propagator);
 /// ```
-pub fn set_http_text_propagator<P: api::HttpTextFormat + Send + Sync + 'static>(propagator: P) {
-    let _lock = GLOBAL_HTTP_TEXT_PROPAGATOR
+pub fn set_http_text_propagator<P: api::TextMapFormat + Send + Sync + 'static>(propagator: P) {
```

Comment:
Think we should probably update this method name as well

Reply:
Whoops. Sorry about that. Fixed.